src/components/ha-climate-control.html

```diff
@@ -84,21 +98,20 @@
     } else {
       this.value = this.min;
     }
+    this.temperatureStateInFlux(true);
+    this.valueChanged();
   }
   valueChanged() {
```

Comment:
The bug was due to the following flow:
* The value is changed to max. last_updated is changed and observer is fired and sets timeout to 2010ms in the future.
* incrementValue is called. last_updated is changed but observer is *not* called.
* 2010ms after the first click callback is called but 2s didn't pass from last click.

To fix this - last_changed should only be changed when value changes.

Reply:
I will try the suggestion

Reply:
You were right, the problem was last_updated. Fixed that now based on the above suggestion.